src/is.test.ts

```diff
@@ -33,16 +33,319 @@ import {
+    expect(isNumberArray(true)).to.equal(false);
+    expect(isNumberArray(false)).to.equal(false);
+    expect(isNumberArray(['String', 2, false])).to.equal(false);
+    expect(isNumberArray([1, 2, 'String', true])).to.equal(false);
+    expect(isNumberArray({})).to.equal(false);
+    expect(isNumberArray()).to.equal(false);
+  });
+});
+
+// isString(value)
+describe('isString(value)', () => {
+  it('Should be a function', () => {
+    expect(isString).to.be.a('function');
+  });
+
+  it('Should return true when "value" is a String', () => {
+    expect(isString('This is a String')).to.equal(true);
+  });
+
+  it('Should return false when "value" is not a String', () => {
+    expect(isString(100)).to.equal(false);
+    expect(isString(undefined)).to.equal(false);
+    expect(isString(null)).to.equal(false);
+    expect(isString(true)).to.equal(false);
+    expect(isString(false)).to.equal(false);
+    expect(isString([])).to.equal(false);
+    expect(isString({})).to.equal(false);
+    expect(isString()).to.equal(false);
+  });
+});
+
+// isStringArray(value)
+describe('isStringArray(value)', () => {
+  it('Should be a function', () => {
+    expect(isStringArray).to.be.a('function');
+  });
+
+  it('Should return true when "value" is type String[]', () => {
+    expect(isStringArray(['String1', 'String2', 'String3'])).to.equal(true);
+    expect(isStringArray(['1', '2', '3'])).to.equal(true);
+    expect(isStringArray([])).to.equal(true);
+  });
+
+  it('Should return false when "value" is not type String[]', () => {
+    expect(isStringArray('This is a String')).to.equal(false);
+    expect(isStringArray(100)).to.equal(false);
+    expect(isStringArray(undefined)).to.equal(false);
+    expect(isStringArray(null)).to.equal(false);
+    expect(isStringArray(true)).to.equal(false);
+    expect(isStringArray(false)).to.equal(false);
+    expect(isStringArray(['String', 2, false])).to.equal(false);
+    expect(isStringArray({})).to.equal(false);
+    expect(isStringArray()).to.equal(false);
+  });
+});
+
+// isInt8Array(value)
+describe('isInt8Array(value)', () => {
+  it('Should be a function', () => {
+    expect(isInt8Array).to.be.a('function');
+  });
+
+  it('Should return true when "value" is an Int8Array', () => {
+    expect(isInt8Array(new Int8Array())).to.equal(true);
   });
 
-  it('Should should return false when "value" is not type any[]', () => {
-    expect(isArray(1)).to.equal(false);
+  it('Should return false when "value" is not an Int8Array', () => {
+    expect(isInt8Array('This is a String')).to.equal(false);
+    expect(isInt8Array(100)).to.equal(false);
+    expect(isInt8Array(undefined)).to.equal(false);
+    expect(isInt8Array(null)).to.equal(false);
+    expect(isInt8Array(true)).to.equal(false);
+    expect(isInt8Array(false)).to.equal(false);
+    expect(isInt8Array([])).to.equal(false);
+    expect(isInt8Array({})).to.equal(false);
+    expect(isInt8Array(new Float64Array())).to.equal(false);
```

Comment:
Add a Uint8Array for good measure:

```suggestion
    expect(isInt8Array(new Float64Array())).to.equal(false);
    expect(isInt8Array(new Uint8Array())).to.equal(false);
```